### PR TITLE
Update apps.json

### PR DESCRIPTION
**Pull request by javadalmasi**
add hicdn

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -4078,6 +4078,26 @@
       ],
       "icon": "HP.svg",
       "website": "http://hp.com/networking"
+    },
+     "HiCDN": {
+      "cats": [
+        31
+      ],
+      "excludes": [
```
[the remark this answers is not in the document]
Comment (javadalmasi) [reply src/apps.json]: We use Cloudflare, arvancloud, backtory and fastly in our services.
Our service is a multiple network that benefits from the above networks, so there is no need to display them all and should be removed.

[the remark this answers is not in the document]
Comment (jvoisin) [reply src/apps.json]: I disagree: the point of wappalyzer is to detect as many technologies as possible. Moreover, if you're using all of those services at the same time, odds are that you're doing things terribly wrong.

[the remark this answers is not in the document]
Comment (jvoisin) [reply src/apps.json]: I don't think that users will go on your website, to see if a site is using cloudflare or not.

In what way would removing the exclusion be confusing?

[the remark this answers is not in the document]
Comment (javadalmasi) [reply src/apps.json]: Your conversation is correct, but those who receive our service are all aware of this, underneath this service is a hybrid network, and those who are providing the service need it and are aware of it.
People who log in through the wappalyzer will be notified.
However, if you do not want to be "excluded" , change it to "implies".

[the remark this answers is not in the document]
Comment (jvoisin) [reply src/apps.json]: Wappalyzer isn't a software tailored for your customers. Speaking of which, you didn't provide any metrics demonstrating HiCDn [qualifies](https://www.wappalyzer.com/docs#new-application) for a inclusion in Wappalyzer.

[the remark this answers is not in the document]
Comment (javadalmasi) [reply src/apps.json]: Our services are in the sub-domain. Check out https://us-3020.cdnit.ir for more information.

[the remark this answers is not in the document]
Comment (javadalmasi) [reply src/apps.json]: I apologize if I've reported to you incorrectly
https://us-3020.cdnit.ir is a specified cdn address for the main domain https://datagharch.com/.
Follow the lines below to get a better idea:
https://www.whatsmydns.net/#A/us-3020.cdnit.ir
https://www.whatsmydns.net/#A/datagharch.com
and also
Use the https://tools.keycdn.com/ping tool to test the two ping addresses of datagharch.com and us-3020.cdnit.ir. You will understand the results.
Due to our privacy policy, we can not add more websites.

[the remark this answers is not in the document]
Comment (jvoisin) [reply src/apps.json]: The links you provided are showing that your CDN has several IP addresses. This isnt' demonstrating that it's a popular one.

**Comment by AliasIO** [comment]
Closing as per @jvoisin's comments.

**Pull request closed without merging**